src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,68 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    public static final String DEFAULT_DELIMITER = ",|:";
+    public static final String CUSTOM_DELIMITER_REGEXP = "//(.)\n(.*)";
+    public static final int BLANK_RESULT = 0;
+
+     public static int splitAndSum(String text){
+
+         if(isBlank(text)) {
+             return BLANK_RESULT;
+         }
+
+         Matcher matcher = Pattern.compile(CUSTOM_DELIMITER_REGEXP).matcher(text);
+         if(matcher.find()){
+             String customDelimiter = matcher.group(1);
+             List<String> tokens = new ArrayList<>();
+             tokens = split(matcher.group(2), customDelimiter);
```

Comment:
```suggestion
             List<String> tokens = split(matcher.group(2), customDelimiter);
```
이렇게 작성하면 불필요한 선언을 하지 않고 코드를 간결하게 사용할 수 있지 않을까요?